GiEnJul/ClientApp/src/App.tsx

```diff
@@ -1,19 +1,41 @@
 import * as React from 'react';
 import { Route } from 'react-router';
-import Layout from './components/Layout';
-import Home from './routes/home/Home';
+import Home from './components/home/Home';
 import Institution from './components/institution/InstitutionMacro';
 import AdminPage from '../src/components/admin/AdminPage';
-import SignUp from './routes/registerGiver/SignUp';
-
+import SignUp from './components/registerGiver/SignUp';
 import './custom.css'
+import NavMenu from './common/components/NavMenu';
+import { Container } from '@material-ui/core';
+import NavMenuAdmin from './common/components/NavMenuAdmin';
+import Edit from './components/admin/editLandingPage/EditMacro';
```

Comment:
Could EditMacro be replaced with EditQuestions? As EditMacros only functionality is returning EditQuestions?

Reply:
The idea was to add different components for editing different sections and let editMacro wrap the sections.